### PR TITLE
R9 Tx, Rx pages

### PR DESCRIPTION
**Pull request by jlpoltrack**
@brad112358  @tmcadam 

As the R9 page has grown rather large, I'm trying to split into separate Tx and Rx hardware pages.  I've done the initial split for the Tx page only - any feedback that you have as I go along would be helpful.

### CODE REVIEW

File: docs/FRSKY_R9_TX.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+Once the ELRS bootloader is installed, the following precedure can be followed to install/update the mLRS firmware.
+
+1. Download the latest mLRS firmware, or build with MLRS\_FEATURE\_ELRS\_BOOTLOADER defined.
+2. Copy the latest tx-R9M-f103c8-elrs-bl-v*.elrs file to FIRMWARE folder on your radio's SD card. Note the "elrs-bl" in the file name.
+   - Note: On bw radios it will be necessary to rename the firmware file to a shorter file name such as "tx-r9m.elrs"
+3. Enter the System Menu and navigate using the page buttons to the SD card page.
+4. Select the FIRMWARE folder and scroll to and select the tx-R9M-f103c8-elrs-bl-v*.elrs file. Choose "Flash external ELRS".
+
+#### Install the mLRS Lua Script ####
```
Comment (olliw42) [inline docs/FRSKY_R9_TX.md:112]: duplicate, is also below
out of sequence too as not on flashing

File: docs/FRSKY_R9_TX.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+### Flash/Update via ST-Link ###
+
+Notes:
+  - Flashing any Frsky R9 module with ST-Link means that it is not possible to revert back to the original Frsky firmware. However, it is possible to switch to ExpressLRS.
+  - In the ExpressLRS docs it is suggested to download and use the "ST-LINK Utility" software. This software is pretty outdated (NRND = not recommended for new designs). Please use the new and recommended tool "STM32CubeProgrammer".
+
+1. Download the non-"elrs-bl" firmware, which can be identified by the ".hex" extension and that the label "elrs-bl" is not contained in the firmware file name. 
+2. Follow the instructions [here](https://www.expresslrs.org/quick-start/transmitters/frsky-r9modules/#via-stlink).
+
+#### Install the mLRS Lua Script ####
```
Comment (olliw42) [inline docs/FRSKY_R9_TX.md:125]: a bit strange here
should it be ## xx ## titled?

File: docs/FRSKY_R9_TX.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+Once mLRS is installed, you can verify that everything is working as expected using the Lua script, instructions are found [here](LUA.md).
+
+### ESP32 Wireless Bridge ###
```
Comment (olliw42) [inline docs/FRSKY_R9_TX.md:129]: should be ## xx ## titled, is no sorted under Flashing
the subsequent sub titles should be ### xx ### titled

File: docs/FRSKY_R9_TX.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+  2. Cut a 5 pin length of the included pin header female connector and pull out the pin from the last position. 
+    - This position serves as a key to avoid plugging in the board incorrectly. 
+  3. Solder the pins in the thru holes as shown below (key position hanging over the left) and reinstall the plastic cover.
+  4. Program the sketch onto the board, detailed below.
+  5. Install on the back of the R9M as shown above. 
+  6. You can then connect your GCS computer or mobile device to the "mLRS AP UDP" WiFi access point and connect the GCS via UDP on port 14550.
+
+<br> <img src="images/Frsky_R9M_M5Stamp_headers.png" width="720px">
+
+### Cooling ###
```
Comment (olliw42) [inline docs/FRSKY_R9_TX.md:167]: should be ## xx ## titled, as now is sorted

File: docs/FRSKY_R9_TX.md (hunk 1 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9_TX.md:112]: Removed

File: docs/FRSKY_R9_TX.md (hunk 2 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9_TX.md:125]: Updated to ##

File: docs/FRSKY_R9_TX.md (hunk 3 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9_TX.md:129]: Updated

File: docs/FRSKY_R9_TX.md (hunk 4 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9_TX.md:167]: Updated

File: docs/FRSKY_R9_TX.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+Once the ELRS bootloader is installed, the following precedure can be followed to install/update the mLRS firmware.
+
+1. Download the latest mLRS firmware, or build with MLRS\_FEATURE\_ELRS\_BOOTLOADER defined.
+2. Copy the latest tx-R9M-f103c8-elrs-bl-v*.elrs file to FIRMWARE folder on your radio's SD card. Note the "elrs-bl" in the file name.
+   - Note: On bw radios it will be necessary to rename the firmware file to a shorter file name such as "tx-r9m.elrs"
+3. Enter the System Menu and navigate using the page buttons to the SD card page.
+4. Select the FIRMWARE folder and scroll to and select the tx-R9M-f103c8-elrs-bl-v*.elrs file. Choose "Flash external ELRS".
+
+#### Flash/Update via ST-Link ####
```
Comment (olliw42) [inline docs/FRSKY_R9_TX.md:112]: should probably be ### titled ?

File: docs/FRSKY_R9_TX.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+      <td>30 dBm (1 W)</td>
+    </tr>
+    <tr>
+      <td>Supported Modes</td>
+      <td>19 Hz</td>
+    </tr>
+  </tbody>
+</table>
+
+Note that R9M module cannot connect with mLRS boards which support 868/915 MHz and use the SX126x LoRa chipset. This is because R9 hardware uses the SX127x LoRa chipset, which is incompatible with the SF6 spreading factor used by the mLRS 19 Hz mode. In addition, the SX127x does not support the SF5 spreading factor which is used for the mLRS 31 Hz mode.
```
Comment (olliw42) [inline docs/FRSKY_R9_TX.md:24]: I wonder if we should go with "our" susual style and write "Note: The R9M...." ?

**Comment by olliw42** [comment]
M A N Y T H X !!!

File: docs/FRSKY_R9_TX.md (hunk 5 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9_TX.md:112]: Updated

File: docs/FRSKY_R9_TX.md (hunk 6 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9_TX.md:24]: Updated

**Comment by olliw42** [comment]
Q: how do you plan to handle the cases where the R9M tx can be used as receiver and the R9 MX rx be used as transmitter?

currently the R9M as receiver option went away from the R9 TX page. Makes sense, but wouldn't then R9 MX as tx have to be listed on that page too?

**Comment by jlpoltrack** [comment]
> R9M tx can be used as receiver

I guess there are 2 options, I prefer 1.

1. Put at bottom of the R9M Transmitter Module Page
2. Add to the Receivers page

> R9 MX rx be used as transmitter

I kind of prefer to not document this option - if we plan to have a statement saying that R9 receivers generally aren't recommended over ELRS hardware then IMO it doesn't make sense to document (I was also a bit surprised to see this in the pre-built FW targets :) )

**Comment by olliw42** [comment]
point 1. makes sense to me

not to document R9MX maybe lesser so ... in contrast to the ELRS receivers it does have two main differences
- it has a 5th pin which can be FULLY used as jrpin5 input
- its button can be used to select the cli at power up

so, while the ELRS receivers cannot be really used as tx modules, the R9MX can be used very well

**Comment by jlpoltrack** [comment]
Stale, so closing.

**Pull request closed without merging**